CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Unreleased
+
+  - Added a CLI method to "ping" the Solr and Source servers for a given 
+    data center.
```

Comment:
Shouldn't `errcode` be documented here, as well?

Reply:
Ah, yes.  I'll add that now.